lib/model/bloc/transactions/TransactionToProfitLossListItemViewModel.dart

```diff
@@ -43,7 +43,7 @@ class TransactionToProfitLossListItemViewModel
   String _title(TransactionEntity from) {
     final dateString =
         (from.timestamp == null) ? "" : _dateFormatter.format(from.timestamp);
-    return dateString + _Strings.divider + from.tag;
+    return dateString + _Strings.divider + (from.tag ?? "");
```

Comment:
If from.tag is null and we are getting the empty string then maybe we should avoid to set divider?

Reply:
👍

Reply:
done